todoCLI/todoCLI.js

```diff
@@ -0,0 +1,94 @@
+const readline = require('readline');
+const fs = require('fs');
+
+const rl = readline.createInterface({
+    input: process.stdin,
+    output: process.stdout,
+    prompt: ">"
+})
+
+
+const toDoMenu = '(v) View • (n) New • (cX) Complete • (dX) Delete • (q) Quit';
+const toDoListArr = [];
+
+function view() {
+    if(toDoListArr.length < 1) console.log('List is empty...');
+    console.log(toDoListArr.join('\n'));
+    console.log(toDoMenu);
+}
+
+function create(task) {
+    toDoListArr.push(`${toDoListArr.length} [] ${task}`);
+    console.log(toDoMenu);
+}
+
+function complete(index) {
+    toDoListArr[index] = toDoListArr[index].replace('[]', '[✓]');
+    console.log(`Completed "${toDoListArr[index].split(']')[1]}"`);
+    console.log(toDoMenu);
+}
+
+function remove(index) {
+    console.log(`Deleted "${toDoListArr[index].split(']')[1]}"`);
+    toDoListArr.splice(index, 1);
+    for (let i = index; i < toDoListArr.length; i++) {
+        toDoListArr[i] = toDoListArr[i].replace(/^\d+/, function(n){ return --n })
+    }
+    console.log(toDoMenu);
+}
+
+function quit(){
+    console.log('See you soon! 😄')
+}
+
+
+console.log('Welcome to Todo CLI!\n--------------------\n' + toDoMenu);
+rl.prompt();
+
+rl.on("line", (line) => {
```

Comment:
just a suggestion, would be nice if you can add a condition to handle a case in which an undefined index is provided.